automations/intake.py

```diff
@@ -1,80 +1,72 @@
 from magicbot import StateMachine, state, timed_state
 from components.intake import Intake
-from automations.lifter import LifterAutomation
 
 
 class IntakeAutomation(StateMachine):
-    """Importing objects from other files"""
     intake: Intake
-    lifter_automation: LifterAutomation
 
     def on_enable(self):
         if not self.intake.arms_out:
             self.engage(initial_state="start")
 
-    @timed_state(must_finish=True, duration=1, next_state="stop")
-    def start(self, state_tm):
-        if state_tm < 0.5:
-            self.intake.extension(True)
-            self.intake.rotate(1)
-            self.intake.arms_out = True
-        else:
-            self.intake.extension(False)
-            self.intake.intake_clamp(False)
-            self.intake.intake_push(False)
+    @timed_state(must_finish=True, duration=0.5, next_state="stop")
+    def start(self):
+        """Get the intake arms out of their starting position."""
+        self.intake.extend(True)
+        self.intake.rotate(1)
+        self.intake.arms_out = True
 
     @state(first=True, must_finish=True)
     def intake_cube(self, state_tm):
-        """Starts the intake motors while waiting for the cube be seen by the
-        infrared sensor"""
-        if self.intake.seeing_cube() and state_tm > 0.5:
-            self.intake.intake_clamp(False)
+        """Start the intake motors and wait for the infrared sensor to see the cube."""
+        if self.intake.is_seeing_cube() and state_tm > 0.5:
+            self.intake.clamp(False)
             self.next_state("pulling_in_cube")
         else:
             self.intake.rotate(-1)
-            self.intake.extension(True)
+            self.intake.extend(True)
 
     @timed_state(must_finish=True, duration=1, next_state="grab_cube")
     def pulling_in_cube(self):
-        self.intake.extension(False)
+        self.intake.extend(False)
         self.intake.rotate(-1)
 
     @state(must_finish=True)
     def grab_cube(self):
-        self.intake.intake_clamp(True)
-        self.intake.intake_push(False)
+        self.intake.clamp(True)
+        self.intake.push(False)
         self.next_state("stop")
 
     @state(must_finish=True)
     def deposit(self):
         """Deposit cube."""
-        if self.intake.seeing_cube():
+        if self.intake.is_seeing_cube():
             self.next_state("push_out_cube")
```

Comment:
Now that I actually know what that `seeing_cube` method was intended to do, this state seems wrong.